New docs for the updated monitor status page (#25615)
DataDog/documentation: content/en/monitors/status/status_page.md

---
title: Monitor Status Page
description: "Get an overview of your monitor status over time"
code_lang: status_page
type: multi-code-lang
weight: 1
aliases:
- /monitors/monitor_status/
- /monitors/manage/status
further_reading:
- link: "/monitors/status/graphs/"
  tag: "Documentation"
  text: "Learn more about monitor graphs"
- link: "/monitors/status/events/"
  tag: "Documentation"
  text: "Learn more about event details"
---

<div class="alert alert-warning">The provisional status page has limited support for monitors and their features. For more details, see <a href="#restrictions-of-provisional-status-page">Restrictions of provisional status page</a>.<br><br>If you are using the legacy status page, see the <a href="/monitors/status/status_legacy">Status Page (Legacy)</a> documentation</div>

## Overview

> When a monitor alerts, the first place to start your investigation is the Monitor Status page. This page displays the graphs and related events to show you why your monitor is in an alert status, so you can understand what is happening around a monitor alert. Further, you can access quick actions to help you move the investigation towards resolution. Use the monitor status page to:

Reviewer: > ... and related events ...

Maybe we should avoid using the word "related" here, since what we mean are only monitor's events.
Users might get confused and understand that "related" events are shown there - e.g. events from a related service

Author: ```suggestion
When a monitor alerts, the first place to start your investigation is the Monitor Status page. This page displays the graphs and events to show you why your monitor is in an alert status, so you can understand what is happening around a monitor alert. Further, you can access quick actions to help you move the investigation towards resolution. Use the monitor status page to:
```

- Review all the context you need to start an investigation
- Investigate the possible cause of an alert
- Take action to escalate, maintain, or resolve your investigation

## Header

{{< img src="/monitors/status/status_page_header.png" alt="Monitor status page header" style="width:100%;" >}}

The header contains the monitor's metadata including:- Alert status
- Monitor status
- Monitor type
- Creation date
- Author
- Associated teams (if available)
- Tags

> - Monitor quality actions

Reviewer: - Monitor quality actions
We might hide this for the moment (sorry for the confusion)

Author: ```suggestion
```

On the right, you'll find the **Edit**, **Clone**, **Export**, and **Permissions** buttons.

From the header, you can resolve the monitor (resolving all groups in alerts) and manage downtimes. For more information, see [Resolve an alert](#resolve-an-alert). 

Additionally, you can view downtimes impacting this monitor and create new ones to mute notifications.

## View monitor evaluations through graphs

 {{< img src="/monitors/status/view_monitor_evaluations_graphs.png" alt="Example monitor evaluation graph, filtered by the prod account template variable." style="width:100%;" >}}

> You can troubleshoot monitor alerts by viewing your data through graphs. Toggle between the **Evaluated Data**, **Source Data**, and **In Groups** graphs to investigate which groups are causing the alert. The graphing component also contains details on the monitor query, configured monitor evaluation, and notification information. To learn more about each graph, see the [Monitor status graphs][1] documentation.

Reviewer: >  and **In Groups** graphs
We went through a quick rename of that section. Now is called "Transitions"

Author: ```suggestion
You can troubleshoot monitor alerts by viewing your data through graphs. Toggle between the **Evaluated Data**, **Source Data**, and **Transitions** graphs to investigate which groups are causing the alert. The graphing component also contains details on the monitor query, configured monitor evaluation, and notification information. To learn more about each graph, see the [Monitor status graphs][1] documentation.
```

Use the template variables to scope down the monitor page to specific groups, select the attributes you want to filter by. 

## View monitor events details

{{< img src="/monitors/status/status_page_event_details.png" alt="Example monitor status event details" style="width:100%;" >}}

You can explore different product areas while maintaining the same alerting context, ensuring you view the same timeframe and service parameters as on your monitor's status page. Use Event details to troubleshoot if the monitor is alerting due to recent configuration changes, take the query to other product areas to troubleshoot and cross reference without losing context. 

For more information, see the [Monitor status events][2] documentation.

> ### Resolve an alert

Reviewer: This entire section ("Resolve an alert") is a bit unexpected to be here

> You can resolve a monitor alert from the Header or the Next Actions sections in the Event details. Resolving from the Header sets the entire monitor to 'OK' (all groups), while resolving from Next Actions only affects the group related to the selected event.

Reviewer: > Resolving from the Header sets the entire monitor to 'OK' (all groups),

Maybe we can be even more explicit and say that when resolving from the header all the groups in alert will be resolved and therefore the monitor status will be to OK

Author: ```suggestion
You can resolve a monitor alert from the Header or the Next Actions sections in the Event details. Resolving from the Header resolves all groups in the alert and sets the monitor status to 'OK' (all groups), while resolving from Next Actions only affects the group related to the selected event.
```

> From Next Actions, click the **Resolve** button to temporarily set the monitor status to 'OK' for its next evaluation. This does not acknowledge the alert or instruct Datadog to ignore it. Use Resolve when data is reported intermittently, such as when a monitor triggers an alert but stops receiving data, preventing it from recovering to 'OK' on its own. The 'resolve' function or the 'Automatically resolve monitor after X hours' option resets the monitor to 'OK'. 

Reviewer: > From Next Actions, click the **Resolve** button to temporarily set the monitor status to 'OK' for its next evaluation.

I got a bit lost when I read this. when the user click on resolve in the next actions (within the event details) only the group is resolved, so it's not sure that the monitor status will be set to ok since there might be other groups in alert and therefore the monitor status will stay in alert.

Reviewer: Maybe we should use what we had [before](https://docs.datadoghq.com/monitors/status/#resolve). At the end nothing changed on the functionality, it works at the same way

---
title: Monitor Status Page
description: "Get an overview of your monitor status over time"
code_lang: status_page
type: multi-code-lang
weight: 1
aliases:
- /monitors/monitor_status/
- /monitors/manage/status
further_reading:
- link: "/monitors/status/graphs/"
  tag: "Documentation"
  text: "Learn more about monitor graphs"
- link: "/monitors/status/events/"
  tag: "Documentation"
  text: "Learn more about event details"
---

<div class="alert alert-warning">The provisional status page has limited support for monitors and their features. For more details, see <a href="#restrictions-of-provisional-status-page">Restrictions of provisional status page</a>.<br><br>If you are using the legacy status page, see the <a href="/monitors/status/status_legacy">Status Page (Legacy)</a> documentation</div>

## Overview

> When a monitor alerts, the first place to start your investigation is the Monitor Status page. This page displays the graphs and events to show you why your monitor is in an alert status, so you can understand what is happening around a monitor alert. Further, you can access quick actions to help you move the investigation towards resolution. Use the monitor status page to:

Reviewer 2: ```suggestion
When a monitor alerts, the first place to start your investigation is the monitor status page. This page displays graphs and events to show you why your monitor is in an alert status, so you can understand what is happening around a monitor alert. Further, you can access quick actions to help you move the investigation towards resolution. Use the monitor status page to:
```

- Review all the context you need to start an investigation
- Investigate the possible cause of an alert
- Take action to escalate, maintain, or resolve your investigation

## Header

{{< img src="/monitors/status/status_page_header.png" alt="Monitor status page header" style="width:100%;" >}}

> The header contains the monitor's metadata including:- Alert status

Reviewer 2: ```suggestion
The header contains the monitor's metadata, including:
- Alert status
```

Reviewer 2: This section seems to switch between using "Header" and "header", capitalize everywhere for consistency?

Author: Nope, just missed it in my review! It should be consistently lowercase :)

- Monitor status
- Monitor type
- Creation date
- Author
- Associated teams (if available)
- Tags

On the right, you'll find the **Edit**, **Clone**, **Export**, and **Permissions** buttons.

> From the header, you can resolve the monitor. Resolving from the Header resolves all groups in the alert and sets the monitor status to 'OK' (all groups). The `resolve` function temporarily changes the monitor status to 'OK' for its next evaluation, but the next evaluation proceeds as normal based on current data.To resolve from the Event details, see [Status Events][1].

Reviewer 2: ```suggestion
From the header, you can resolve the monitor. Resolving from the Header resolves all groups in the alert and sets the monitor status to 'OK' (all groups). The `resolve` function temporarily changes the monitor status to 'OK' for its next evaluation, but the next evaluation proceeds as normal based on current data. To resolve from the Event details, see [Status Events][1].
```

Use the template variables to scope down the monitor page to specific groups, select the attributes you want to filter by. 

## View monitor events details

{{< img src="/monitors/status/status_page_event_details.png" alt="Example monitor status event details" style="width:100%;" >}}

You can explore different product areas while maintaining the same alerting context, ensuring you view the same timeframe and service parameters as on your monitor's status page. Use Event details to troubleshoot if the monitor is alerting due to recent configuration changes, take the query to other product areas to troubleshoot and cross reference without losing context. 

For more information, see the [Monitor status events][3] documentation.

## Restrictions of provisional status page 

### Limited support for monitor types

The following monitor types are not supported by the provisional status page:

> - Event  
> - Metrics  
> - Log

Reviewer: ```suggestion
- Anomaly
- Cloud Cost
- Composite
- Database Monitoring
- Forecast
- Live Process
- Outlier
- Synthetics
- SLO Alerts
- Usage
```

- Review all the context you need to start an investigation
- Investigate the possible cause of an alert
- Take action to escalate, maintain, or resolve your investigation

## Header

{{< img src="/monitors/status/status_page_header.png" alt="Monitor status page header" style="width:100%;" >}}

The header contains the monitor's metadata, including:
- Alert status
- Monitor status
- Monitor type
- Creation date
- Author
- Associated teams (if available)
- Tags

On the right, you'll find the **Edit**, **Clone**, **Export**, and **Permissions** buttons.

From the header, you can resolve the monitor. Resolving from the header resolves all groups in the alert and sets the monitor status to `OK` (all groups). The `resolve` function temporarily changes the monitor status to `OK` for its next evaluation, but the next evaluation proceeds as normal based on current data. To resolve from the Event details, see [Status Events][1].

> Additionally, view and manage downtimes impacting this monitor and create new ones to mute notifications. 

Reviewer: Maybe here we could add a screenshot like the one below to clarify how to check donwtimes impacting this monitor
<img width="1918" alt="image" src="https://github.com/user-attachments/assets/8cfb6860-0f66-4677-a9ae-ade4d5739cbe">
()

{{< img src="/monitors/status/header_downtimes.png" alt="The monitor is not muted, but you can click Not Muted to see options to create a downtimes for this monitor" style="width:100%;" >}}

Additionally, view and manage downtimes impacting this monitor and create new ones to mute notifications. 

## View monitor evaluations through graphs

 {{< img src="/monitors/status/evaluated_data_graph_1.png" alt="Example monitor evaluation graph" style="width:100%;" >}}

You can troubleshoot monitor alerts by viewing your data through graphs. Toggle between the **Evaluated Data**, **Source Data**, and **Transitions** graphs to investigate which groups are causing the alert. The graphing component also contains details on the monitor query, configured monitor evaluation, and notification information. To learn more about each graph, see the [Monitor status graphs][2] documentation.

Use the template variables to scope down the monitor page to specific groups, select the attributes you want to filter by. 

## View monitor events details

{{< img src="/monitors/status/status_page_event_details.png" alt="Example monitor status event details" style="width:100%;" >}}

> You can explore different product areas while maintaining the same alerting context, ensuring you view the same timeframe and service parameters as on your monitor's status page. Use Event details to troubleshoot if the monitor is alerting due to recent configuration changes, take the query to other product areas to troubleshoot and cross reference without losing context. 

Reviewer 2: ```suggestion
You can explore different product areas while maintaining the same alerting context, ensuring you view the same timeframe and service parameters as on your monitor's status page. Use Event details to troubleshoot if the monitor is alerting due to recent configuration changes, and take the query to other product areas to troubleshoot and cross reference without losing context. 
```
Or just break it into separate sentences?
